manifests/README.md

```diff
@@ -10,6 +10,10 @@ The operating system must provide:
 Consult the documentation of your Linux distribution on details for enabling these prerequisites.
 ## Installing Kepler on Kubernetes
 
+Simple option:
+
+For people who just want a quick try:
+
 ```bash
 # NOTE: These manifest take care of creating the namespace and Kepler exporter
 kubectl apply -f kubernetes/deployment.yaml
```

Comment:
is this path still valid?

Reply:
technicaly yes, as there is a symlink to base (deployment.yaml -> base/deployment.yaml) and  base/deployment.yaml
still useable for now

the reason for doing this is to make things simple(so first time user has less step to perform), but I think sooner or later we might need run kustomize inevitable